### PR TITLE
[Enhancement] Summary restyling

### PR DESCRIPTION
**Pull request by francofusaro**
Summary restyling according to https://www.figma.com/file/9qqumGDOn9516LScLVPAfZ/%5BPX%5D-Restyling-Summary?node-id=93%3A3237

Contains the changes of this PRs:
- ["Como"](https://github.com/mercadopago/px-android/pull/2559)
- ["Cuanto"](https://github.com/mercadopago/px-android/pull/2557)
- [Offline methods](https://github.com/mercadopago/px-android/pull/2563)
- [Adjustments](https://github.com/mercadopago/px-android/pull/2560)
 
Plus the changes for "Quien" (thumbnail, title and subtitle)

## Screenshots
<div style="display:flex">

</div>

### CODE REVIEW

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/animations/ExpandAndCollapseAnimation.java [hunk 1]
```diff
@@ -53,4 +53,8 @@ public void collapse() {
             targetView.setVisibility(View.GONE);
         }
     }
+
+    public boolean shouldCollapse() {
```
Comment (cgaggino) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/animations/ExpandAndCollapseAnimation.java:57]: para que se agregó esto? :thinking:

Comment (cgaggino) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/animations/ExpandAndCollapseAnimation.java:57]: No dije nada ya lo vi

Comment (francofusaro) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/animations/ExpandAndCollapseAnimation.java:57]: para fixear el bug de las cuotas v1 que se colapsaban solas aunque no estuvieran abiertas

Comment (francofusaro) [reply px-checkout/src/main/java/com/mercadopago/android/px/internal/features/express/animations/ExpandAndCollapseAnimation.java:57]: (cuando usabas split digo)

File: px-checkout/src/main/java/com/mercadopago/android/px/internal/view/ToolbarCancel.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import android.util.AttributeSet
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
+import androidx.core.content.ContextCompat
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.graphics.BlendModeColorFilterCompat
+import androidx.core.graphics.BlendModeCompat
+
+class ToolbarCancel @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) :
```
Comment (cgaggino) [inline px-checkout/src/main/java/com/mercadopago/android/px/internal/view/ToolbarCancel.kt:21]: podria ser internal?

**Review by cgaggino — APPROVED** [review]

**Pull request merged**